libvast/vast/transform.hpp

```diff
@@ -78,6 +81,10 @@ class transformation_engine {
   transformation_engine() = default;
   explicit transformation_engine(std::vector<transform>&&);
 
+  /// Returns an error if any of the transforms is an aggregate and
+  /// aggregates are not allowed. //FIXME: use enum
```

Comment:
:+1: on using an `enum class` for this, e.g.,

```cpp
/// docs
enum class allow_aggregate_transforms {
  yes, ///< docs
  no,  ///< docs
}
```

Reply:
What is your opinion about:

```
enum aggregate_validation { allow_aggregates, forbid_aggregates };
```

Reply:
We use the yes/no approach in other places already, so I recommend using that for consistency.